src/cable/websocket_pinger.cr

```diff
@@ -25,12 +23,12 @@ module Cable
 
     def initialize(@socket : HTTP::WebSocket)
       @task = Tasker.every(Cable::WebsocketPinger.seconds.seconds) do
-        raise PingStoppedException.new("Stopped") if @socket.closed?
+        next stop if @socket.closed?
```

Comment:
exit the block and call `stop` to ensure the tasker is cleaned up if the socket is closed